common/app/templates/headerInlineJS/bootSystemJS.scala.js

```diff
@@ -49,10 +49,21 @@ System['import']('core').then(function () {
             // Report uncaught exceptions
             raven.install();
 
+            var oldOnError = window.onerror;
+            window.onerror = function (message, filename, lineno, colno, error) {
+                // Not all browsers pass the error object
+                if (!error || !error.reported) {
+                    oldOnError.apply(window, arguments);
```

Comment:
Some older browsers don't pass the error object into `window.onerror`, in which cause we will could be reporting the error twice. That's the price we pay for catching and re-throwing errors to gain extra metadata.